orchestrators/Orchestrator.py

```diff
@@ -7,19 +7,19 @@
 from dotenv import load_dotenv
 
 class Orchestrator(ABC):
-    load_dotenv()
-
-    DEBUG = os.environ.get("DEBUG", "false")
-    DEBUG_LOGGING = DEBUG.lower() == "true"
-
     @abstractmethod
-    def conversation_with_data(self, request_body):
+    def conversation_with_data(self, request_bod, message_uuidy):
         pass
 
     @abstractmethod
-    def conversation_without_data(self, request_body):
+    def conversation_without_data(self, request_body, message_uuid):
         pass
 
+    load_dotenv()
```

Comment:
This shouldn't be necessary as it already runs in app.py at startup.

Reply:
My two cents - let's get your PR in, I'll pull into mine, and let's think about those env vars after. I think @tendau  had some thoughts on that.